docs/SPEC.md

```diff
@@ -311,9 +310,9 @@ To include the literal curly brackets within a string they must be escaped.
 Interpolation example:
 
     n = 42
-    "the answer is {n}" // the answer is 42
-    "the answer is not {n+1}" // the answer is not 43
-    "openinng curly bracket \{" // openinng curly bracket {
+    "the answer is ${n}" // the answer is 42
+    "the answer is not ${n+1}" // the answer is not 43
+    "opening curly bracket \${" // opening curly bracket ${
     "closing curly bracket \}" // closing curly bracket }
```

Comment:
This line should be removed now, I missed that on the first pass.

Reply:
I kept it mostly because it's a demonstration of the escape sequence, but I'll update the text to make clear the dollar sign is there too.

Reply:
Never mind you meant the closing bracket.